wgpu-types/src/lib.rs

```diff
@@ -560,6 +560,13 @@ bitflags::bitflags! {
         ///
         /// This is a native-only feature.
         const TEXTURE_COMPRESSION_ASTC_HDR = 1 << 42;
+        /// Allows shaders to acquire the FP16 ability
+        /// Supported Platforms:
+        /// - Vulkan
+        /// - Metal
+        ///
+        /// This is a web and native feature.
+        const SHADER_FLOAT16 = 1 << 43;
```

Comment:
This should be in the web section, not in the native section

Reply:
Thanks for pointing out, I was totally unaware of this.